### ISSUE TITLE
There should be a consistent way for users to get to settings from within an extension experience

### ISSUE THREAD

**Issue by joadoumie** [zadjii-msft/PowerToys#195]
For example, if we are in a list page for an extension, we should have some consistent way for users to navigate to whatever settings that extension provides.

**Comment by zadjii-msft** [comment]
from #313

As noted from @crutkas in the self-host... we should make it a bit easier to navigate to settings from the root UI. We should also make it easier to navigate to settings for extensions. We can make this a very consistent experience by allowing the user to click on the bottom-left corner of the UI where we display the current provider, something similar to: 

![image](https://github.com/user-attachments/assets/fc6668f8-5a11-421c-85af-dfb80da5309e)

@niels9001 - this is another UX exploration. The idea is that we would have a menu that allows folks to go to the settings for that provider.

**Comment by niels9001** [comment]
Yeah I like that! Having a settings button in the flyout (next to the infobars) should work?
